### PR TITLE
Add func to run commands on host after scenario

### PR DESCRIPTION
**Pull request by kasturinarra**
Add func to run commands on host after scenario

### CODE REVIEW

**Comment by kasturinarra** [comment]
@geliu2016 could you please help review this ? This func has been used in the script here: https://github.com/openshift/cucushift/pull/8427, thanks !!

**Comment by geliu2016** [comment]
LGTM，thanks

**Comment by kasturinarra** [comment]
@liangxia could you please help review & merge ? Thanks !!

**Comment by kasturinarra** [comment]
@liangxia could you please help review & merge? Thanks !!

File: features/step_definitions/node.rb [hunk 1]
```diff
@@ -140,6 +140,19 @@
   cb[tmpcb] = nil
 end
 
+Given /^I run( background)? commands on the nodes in the#{OPT_SYM} clipboard after scenario:$/ do |bg, cbname, table|
+  ensure_admin_tagged
```
Comment (liangxia) [inline features/step_definitions/node.rb:144]: Perhaps we should add `ensure_destructive_tagged` ?
We can allow to run non-destructive ones in destructive mode, but we should not allow to run destructive ones in non-destructive mode.

**Comment by kasturinarra** [comment]
@liangxia have resolved all comments, could you please help review again ? Thanks !!

**Pull request merged**